### PR TITLE
Add check-run HTML URL to JSON output.

### LINKED ISSUE #310: Include check URL in JSON output
Downstream steps process the JSON output of this action. We'd like them to be able to link back to the created check, but (as far as I can tell) there's no way for them to get that information. It'd be convenient if the link to the check was included in the action's JSON output.

### PR DESCRIPTION
**Pull request by danxmoran**
Closes #310

### CODE REVIEW

File: python/test/test_publisher.py [hunk 1]
```diff
@@ -1315,7 +1315,7 @@ def do_test_publish_check_without_base_stats(self, errors: List[ParseError], ann
                            '3vULqr9z48alUDTb1XTaLTRPlI4YQM0EU2gdwCzIEYwjllAq2P'
                            '1sIUrxjpD1E+YjA0QXwf0TM7hqlgOC5HMP/dt/RevnK18F3THx'
                            'FS08fz1s0zBZBc2w5zHdX73QAAAA==\n'.format(errors='{} errors\u2004\u2003'.format(len(errors)) if len(errors) > 0 else ''),
-                'annotations': annotations
+                'annotations': annotations,
```
Comment (EnricoMi) [inline python/test/test_publisher.py:1318]: This `,` is not needed, right? Can it be removed?

**Review by EnricoMi — APPROVED** [review]
LGTM! Just a minor formatting thing.

I have created PR [#1](https://github.com/danxmoran/publish-unit-test-result-action/pull/1) in your repo. If you activate GitHub actions for that repo, your next commit will run the CI and actually execute the action on that PR. That is the best way to test this. No need to merge that into your master. I cannot test it in my repo before merging.

File: python/test/test_publisher.py (hunk 1 above)
Comment (danxmoran) [reply python/test/test_publisher.py:1318]: Yes I think it's left over from unrelated changes, will revert

Comment (danxmoran) [reply python/test/test_publisher.py]: Done

**Comment by danxmoran** [comment]
@EnricoMi tests are running on https://github.com/danxmoran/publish-unit-test-result-action/pull/1 now, thanks for the pointers!

**Comment by EnricoMi** [comment]
Thanks for the contribution!

**Pull request merged**